### PR TITLE
feat: Add lint

### PR DESCRIPTION
**Pull request by DukeBaird**
- Adds eslint and fixes lint issues
- Adds lint to checks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Ahmic Lake Wildlife Tracker",
   "main": "server.js",
   "scripts": {
+    "lint": "eslint .",
```
Comment (mcraig567) [inline package.json:7]: This file is supposed to change?

File: server/models/Sighting.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-    animal: String,
-    time: Date,
-    location: String,
-    spottedBy: String
+	// _id: String,
+	animal: String,
+	time: Date,
+	location: String,
+	spottedBy: String
 });
```
Comment (mcraig567) [inline server/models/Sighting.js:9]: Are these tabs different sizes? Not sure what it's changing

File: server/index.js [hunk 3]
```diff
@@ -36,35 +34,20 @@ function start() {
...
 
 	// passport is for user auth
 	// require('./lib/passport.js')(passport);
 
-	app.listen(app.get('port'), function() {
-		console.log('Server running on localhost:' + app.get('port'));
+	app.listen(app.get('port'), () => {
+		console.log(`Server running on localhost:${app.get('port')}`);
 	});
 }
```
Comment (mcraig567) [inline server/index.js:52]: Are all these lint/formatting changes? No unused variables, etc?

File: package.json (hunk 1 above)
Comment (DukeBaird) [reply package.json:7]: Yeah, this is adding in a linter, which does code style stuff. Making sure we're consistent across all our files

File: server/models/Sighting.js (hunk 2 above)
Comment (DukeBaird) [reply server/models/Sighting.js:9]: It was probably spaces before, and should be tabs now

File: server/index.js (hunk 3 above)
Comment (DukeBaird) [reply server/index.js:52]: Yeah, this was all lint fixes

**Review by mcraig567 — APPROVED** [review]
Ready for it to scream at me for formatting changes!

**Pull request merged**